code/controllers/subsystem/air.dm

```diff
@@ -65,7 +65,7 @@ Class Procs:
 	name          = "Air"
 	init_order    = SS_INIT_AIR
 	priority      = SS_PRIORITY_AIR
-	wait          = 1 SECONDS
+	wait          = 2 SECONDS
```

Comment:
I don't want to include gameplay changes with system changes. This is what it's set to on live right now.